Octokit/Models/Response/StringEnum.cs

```diff
@@ -29,7 +29,9 @@ public StringEnum(TEnum parsedValue)
 
         public StringEnum(string stringValue)
         {
-            _stringValue = stringValue ?? string.Empty;
+            Ensure.ArgumentNotNull(stringValue, nameof(stringValue));
```

Comment:
should we enforce no empty strings here too?

Reply:
Yeah, I guess it wouldn't hurt.